src/native/libs/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -187,6 +189,12 @@ SSL_CTX* CryptoNative_SslCtxCreate(const SSL_METHOD* method)
                 return NULL;
             }
         }
+
+        // Opportunistically request the server present a stapled OCSP response.
+        if (SSL_CTX_ctrl(ctx, SSL_CTRL_SET_TLSEXT_STATUS_REQ_TYPE, TLSEXT_STATUSTYPE_ocsp, NULL) != 1)
```

Comment:
Is __1__ the documented "success" return value for this function given this parameter? I can't find any docs on the *SSL_CTRL_SET_TLSEXT_STATUS_REQ_TYPE* parameter.

Reply:
The docs play a bit of a chameleon game here.  They call it [SSL_CTX_set_tlsext_status_type](https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_tlsext_status_type.html), where 0 and 1 are the only defined return values.